spec.html

```diff
@@ -1188,6 +1188,31 @@ <h1>
           <p>This algorithm always returns -1 if _fromIndex_ + the length of _searchValue_ > the length of _string_.</p>
         </emu-note>
       </emu-clause>
+
+      <emu-clause id="sec-stringlastindexof" type="abstract operation">
+        <h1>
+          StringLastIndexOf (
+            _string_: a String,
+            _searchValue_: a String,
+            _fromIndex_: a non-negative integer,
+          ): an integer
```

Comment:
```suggestion
          ): either a non-negative integer or -1
```

We'll come through with a follow-up PR after this one to replace `-1` both here and in `StringIndexOf` with a `~not-found~` sentinel. Unless you want to do it, of course.

Reply:
We don't use "either" after a colon (except when needed to disambiguate). See https://github.com/tc39/ecma262/pull/2972#issuecomment-1372894434